apis/livepeer/livepeer.go

```diff
@@ -383,36 +376,27 @@ func (lapi *API) DeleteStream(id string) error {
 
 // CreateStreamEx creates stream with specified name and profiles
 func (lapi *API) CreateStreamEx(name string, record bool, presets []string, profiles ...Profile) (*CreateStreamResp, error) {
```

Comment:
Curious how come this func is named CreateStreamEx, what does `Ex` mean or what is the naming convention here?

Reply:
I don't really know TBH, it is before my time. I could only guess it that means something like "extended"? Like we first had `CreateStream` but then wanted some extension of it with additional capabilities. I know that the `R` suffix means "with retries", so you might find some of those with that.